OMEdit/OMEditGUI/MainWindow.cpp

```diff
@@ -3867,6 +3888,28 @@ void MainWindow::createMenus()
   pToolsMenu->addAction(mpOpenTerminalAction);
   pToolsMenu->addSeparator();
   pToolsMenu->addAction(mpOptionsAction);
+  // Plugins View Menu
+  mpPluginsMenu = new QMenu(menuBar());
+  mpPluginsMenu->setObjectName("PluginsMenu");
+  mpPluginsMenu->setTitle(tr("Plugins"));
+  // load OMSens plugin
+#ifdef Q_OS_WIN
+  QPluginLoader loader("../../../OMSensPlugin/bin/OMSensPlugin.dll");
+#elif defined(Q_OS_MAC)
+  QPluginLoader loader("../../../OMSensPlugin/bin/libOMSensPlugin.dylib");
+#else
+  QPluginLoader loader("../../../OMSensPlugin/bin/libOMSensPlugin.so");
+#endif
```

Comment:
Instead of asking the user to load plugin, I changed it to load the plugin automatically. Once the actual OMSens plugin is placed in the OpenModelica repos then we need to update these links. Right now they are using a dummy [OMSensPlugin](https://github.com/adeas31/OMSensPlugin) which just shows how to implement and use the interfaces.